guide/src/guide/getting_started.md

```diff
@@ -90,7 +92,7 @@ Now we can finally start writing an actual GPU kernel.
 <details>
   <summary>Expand this section if you are not familiar with how GPU-side CUDA works</summary>
 
-Firstly, we must explain a couple of things about GPU kernels, specifically, how they are executed. GPU Kernels (functions) are the entry point for executing anything on the GPU, they are the functions which will be executed from the CPU. GPU kernels do not return anything, they write their data to buffers passed into them.
+Firstly, we must explain a couple of things about GPU kernels, specifically, how they are executed. GPU Kernels (functions) are the entry oint for executing anything on the GPU, they are the functions which will be executed from the CPU. GPU kernels do not return anything, they write their data to buffers passed into them.
```

Comment:
🤔

Reply:
whoops must have deleted that somehow. Thanks for pointing it out. Should be fixed now